AgentInterfaces/VmNetworkConfiguration.cs

```diff
@@ -23,6 +23,11 @@ public class VmNetworkConfiguration
 
     public class Endpoint
     {
+        /// <summary>
+        /// The frontend IP address for the endpoint.
+        /// </summary>
+        public string FrontendIpAddress { get; set; }
```

Comment:
where is this used?

Reply:
This will be used in the VMAgent to get the different endpoints. It was not used before, but I will from now on

Reply:
Where is this gonna used? Just curious

Reply:
I had a same question, Jota mentioned that it will be used in VM Agent, maybe we should add a comment?